plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/BaseClickHouseTypeMapping.java

```diff
@@ -451,10 +451,45 @@ public void testClickHouseChar()
                 .addRoundTrip("char(32)", "'攻殻機動隊'", VARBINARY, "to_utf8('攻殻機動隊')")
                 .addRoundTrip("char(1)", "'😂'", VARBINARY, "to_utf8('😂')")
                 .addRoundTrip("char(77)", "'Ну, погоди!'", VARBINARY, "to_utf8('Ну, погоди!')")
+                .addRoundTrip("char(255)", "''", VARBINARY, "X''")
```

Comment:
Could you add a helper method instead of repeating the same roundtrip?

Reply:
Also done.